README.md

```diff
@@ -113,7 +113,7 @@ Standalone NuGet packages can be configured the same way as SonarLint in connect
 
 You can see how we do it in our own repository for SonarLint Connected mode (and you can reuse the same concept for the NuGet package):
 
-* We reference in each production-code project this [Directory.Build.props](analyzers/src/Directory.Build.props).
+* We reference in each production-code project this [Directory.Build.targets](analyzers/src/Directory.Build.targets).

```

Comment:
related to the https://github.com/SonarSource/sonar-dotnet/pull/5138 refactoring